coverage.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/sh
```

Comment:
I would if not already latest from master, take a pull and move this file to the `scripts` folder :) after all that is the right place for these.